.github/workflows/build-relayer.yaml

```
@@ -0,0 +1,58 @@
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PUSH_TOKEN }}
+
+      # Configure tag name
+      - name: Sets env vars
+        run: |
+          echo "DOCKER_IMAGE_NAME=${{ secrets.DOCKER_USERNAME }}/${{ github.event.repository.name }}-relayer:${GITHUB_SHA:0:8}" >> $GITHUB_ENV
+
+      # Build docker image
+      - name: Build docker image
+        uses: docker/build-push-action@v4
```

actionlint (1.7.8)

the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)